alpha/apps/kaltura/lib/kUploadTokenMgr.php

```diff
@@ -215,25 +248,31 @@ protected function tryMoveToErrors($fileData)
 	 * @param bool $finalChunk        	
 	 * @param float $resumeAt        	
 	 */
-	protected function handleResume($fileData, $finalChunk, $resumeAt)
+	protected function handleResume($fileData, $resumeAt)
 	{
 		$uploadFilePath = $this->_uploadToken->getUploadTempPath();
-		if (! file_exists($uploadFilePath))
+		if (!file_exists($uploadFilePath))
 			throw new kUploadTokenException("Temp file [$uploadFilePath] was not found when trying to resume", kUploadTokenException::UPLOAD_TOKEN_FILE_NOT_FOUND_FOR_RESUME);
 		
 		$sourceFilePath = $fileData['tmp_name'];
 		
 		if ($resumeAt != -1) // this may not be a sequential chunk added at the end of the file
 		{
-			// support backwards compatiblity of overriding a final chunk at the offset zero  
-			$verifyFinalChunk = $finalChunk && $resumeAt > 0;
+			// support backwards compatibility of overriding a final chunk at the offset zero  
+			$verifyFinalChunk = $this->_finalChunk && $resumeAt > 0;
 			
 			// if this is the final chunk the expected file size would be the resume position + the last chunk size
 			$chunkSize = filesize($sourceFilePath);
 			$expectedFileSize = $verifyFinalChunk ? ($resumeAt + $chunkSize) : 0;
 
 			$chunkFilePath = "$uploadFilePath.chunk.$resumeAt";
-			rename($sourceFilePath, $chunkFilePath);
+			$succeeded = rename($sourceFilePath, $chunkFilePath);
```

Comment:
use kFIle::move or add rename to kFile since we are trying to push all file system access via kFIle

Reply:
Will do this as part of a different pull request.